conf/modules.config

```diff
@@ -666,6 +661,42 @@ process {
         ]
     }
 
+    withName: TIARA_TIARA {
+        publishDir = [
+            [
+                path: { "${params.outdir}/GenomeBinning/Tiara" },
+                mode: params.publish_dir_mode,
+                pattern: { "${meta.assembler}-${meta.id}.tiara.{txt}" }
+            ],
+            [
+                path: { "${params.outdir}/GenomeBinning/Tiara/log" },
+                mode: params.publish_dir_mode,
+                pattern: { "log_${meta.assembler}-${meta.id}.tiara.{txt}" }
+            ]
+        ]
+        ext.args = { "--min_len ${params.tiara_min_length} --probabilities" }
+        ext.prefix = { "${meta.assembler}-${meta.id}.tiara" }
+    }
+
+    withName: TIARA_CLASSIFY {
+        publishDir = [
+            path: { "${params.outdir}/GenomeBinning/Tiara" },
+            mode: params.publish_dir_mode,
+            pattern: "*.binclassification.tsv"
+        ]
+        ext.args = { "--join_prokaryotes --assembler ${meta.assembler}" }
+        ext.prefix = { "${meta.assembler}-${meta.binner}-${meta.bin}-${meta.id}" }
+    }
```

Comment:
Maybe make this optional (and by default turned off) given you have `TIARA_SUMMARY` vthat presumably has all the info for you just in one place.

Reply:
Good point - might suggest this is actually entirely unnecessary to publish these files as tiara_summary.tsv should just be all of these concatenated together.